index.js

```diff
@@ -212,55 +214,51 @@ function expandGlob(args) {
   return args;
 }
 
+function explore(code, map, options) {
+  if(typeof options === 'undefined') {
+    if(typeof map === 'object' && !Buffer.isBuffer(map)) {
+      options = map;
+      map = undefined;
```

Comment:
could just make `sourcemap` an `option` to simplify this code.

Reply:
My intention was to be similar to the binary version. You execute it as 
```
source-map-explorer bundle.min.js bundle.min.js.map
```
and you call it as
```
explore('bundle.min.js', 'bundle.min.js.map');
```